packages/iTwinUI-react/src/core/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -23,6 +23,69 @@ export type BreadcrumbsProps = {
    * Defaults to the `SvgChevronRight` icon.
    */
   separator?: React.ReactNode;
+  /**
+   * If specified, this prop will be used to show a custom button when overflow happens,
+   * i.e. when there is not enough space to fit all the breadcrumbs.
+   *
+   * Expects a function that takes the number of items that are visible
+   * and returns the `ReactNode` to render.
+   *
+   * @example <caption>Uses the overflow button to redirect to the previous breadcrumb</caption>
+   * <Breadcrumbs
+        overflowButton={(visibleCount: number) => (
+          <IconButton
+            style={{ paddingTop: '8px' }}
+            onClick={() => {
+              const previousBreadcrumb =
+                visibleCount > 1
+                  ? items.length - visibleCount
+                  : items.length - 2;
+              action(`Visit breadcrumb ${previousBreadcrumb}`)();
```

Comment:
Remove `action` and `args` from both examples cause they are from storybook.

Reply:
Updated